KomaMRIBase/src/datatypes/sequence/ADC.jl

```diff
@@ -96,11 +96,8 @@ end
 """
     phase = get_adc_phase_compensation(seq)
 
-Returns the array of phases for every acquired sample in the sequence `seq`.
-
-!!! note
-    This function is useful to compensate the phase when the RF pulse has a phase too. Refer
-    to the end of the [`run_sim_time_iter`](@ref) function to see its usage.
+Returns the array of phases for every acquired sample in the sequence `seq`. This function
```

Comment:
It does not return an array of phases, it returns a compensation factor  $\exp(-\mathrm{i}\varphi)$, which is used to compensate the acquired signal $S$ by doing $S_{\mathrm{comp}} = S \exp(-\mathrm{i}\varphi)$ (applied after the simulation). This is due to the fact that the signal has a phase offset of $\varphi$ after an RF excitation with phase $\varphi$. These pulses are commonly used in sequences with RF spoiling.